ci/crossbeam-epoch-loom.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"/../crossbeam-epoch
+set -ex
+
+export RUSTFLAGS="-D warnings --cfg=loom_crossbeam"
+
+env LOOM_MAX_PREEMPTIONS=2 cargo test --test loom --features sanitize --release -- --nocapture
```

Comment:
I'd like to ask if you could write here why LOOM_MAX_PREEMPTIONS=2 is good enough.  (Or =3 is too costly.)